src/api/app/views/webui2/webui/package/binaries.html.haml

```diff
@@ -0,0 +1,51 @@
+.card
+  = render partial: 'tabs', locals: { package: @package, project: @project }
+  .card-body
+    %h3 State of Repository #{@repository} for #{@project} / #{@package_name}
+    %p
+      - if @configuration['download_url']
+        = render partial: 'shared/download_repository_link', locals: { project: @project, repository: @repository,
+                                                                       download_url: @configuration['download_url'] }
+      - unless @buildresults.empty?
+        = link_to package_wipe_binaries_path(project: @project, package: @package_name, repository: @repository), method: :delete do
+          %i.fas.fa-times-circle.text-danger
+          Delete all built binaries
+
+    - @buildresults.each do |result|
+      %h3
+        = repo_status_icon(result[:repocode])
+        Architecture: #{result[:arch]}
+      - if result[:binaries].empty?
+        %p
+          %i No built binaries
+      - else
+        .table-responsive
+          %table.table.table-hover
+            %tbody
+            - result[:binaries].each do |binary|
+              %tr
+                %td.text-nowrap #{binary[:filename]} (#{number_to_human_size(binary[:size])})
+                %td.text-nowrap
+                  - if binary[:links][:cloud_upload?]
+                    = link_to(new_cloud_upload_path(project: @project, package: @package, repository: @repository, arch: result[:arch],
+                    filename: binary[:filename]), title: 'Cloud Upload') do
+                      %i.fas.fa-cloud-upload-alt
+                      Cloud Upload
+                  - if binary[:links][:download_url]
+                    = link_to(binary[:links][:download_url], title: 'Download') do
+                      %i.fas.fa-arrow-circle-down.text-secondary
```

Comment:
This needs to be like `Download` [here](https://obs-patterns.netlify.com/content/icons.html)